schemas/common_definitions.json

```diff
@@ -11,6 +11,11 @@
     "description": "Used to identify groups, blocks, questions and answers.",
     "pattern": "^[a-z0-9][a-z0-9\\-]*[a-z0-9]$"
   },
+  "metadata_identifier": {
+    "type": "string",
+    "description": "Used to identify groups, blocks, questions and answers.",
```

Comment:
This description needs changing, however this is now duplicated in 
https://github.com/ONSdigital/eq-questionnaire-validator/blob/master/schemas/when_rule/definitions.json#L35-L38

It should live in commands and be shared.